### ISSUE TITLE
EventException caused by NPE

### ISSUE THREAD

**Issue by SlimeDog** [PikaMug/Quests#760]
**What is your Quests version / build number (do _not_ say "latest")?**
Spigot 1.13.2
Citizens 2.0.24-b1628
Sentinel 1.8.5-b243
Quests 3.6.5-b716

**Describe the problem and how to replicate it. Any console errors to post _using Pastebin.com_?**
Test server, one player on-line.
Sentinel dogs are killing auto-generated bats.

Frequent console spam:
```
[09:35:42 ERROR]: Could not pass event EntityDeathEvent to Quests v3.6.5-b716
org.bukkit.event.EventException: null
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:316) ~[spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:66) ~[spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:507) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:492) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callEntityDeathEvent(CraftEventFactory.java:573) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.EntityLiving.die(EntityLiving.java:1205) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.EntityLiving.damageEntity(EntityLiving.java:1055) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.EntityBat.damageEntity(EntityBat.java:169) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at org.bukkit.craftbukkit.v1_13_R2.entity.CraftLivingEntity.damage(CraftLivingEntity.java:227) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at org.mcmonkey.sentinel.SentinelWeaponHelper.punch(SentinelWeaponHelper.java:171) [Sentinel-1.8.5-b243.jar:?]
        at org.mcmonkey.sentinel.SentinelAttackHelper.tryAttackInternal(SentinelAttackHelper.java:359) [Sentinel-1.8.5-b243.jar:?]
        at org.mcmonkey.sentinel.SentinelAttackHelper.tryAttack(SentinelAttackHelper.java:63) [Sentinel-1.8.5-b243.jar:?]
        at org.mcmonkey.sentinel.SentinelTrait.runUpdate(SentinelTrait.java:1081) [Sentinel-1.8.5-b243.jar:?]
        at org.mcmonkey.sentinel.SentinelTrait.run(SentinelTrait.java:1290) [Sentinel-1.8.5-b243.jar:?]
        at net.citizensnpcs.api.npc.AbstractNPC.update(AbstractNPC.java:428) [Citizens-2.0.24-b1628.jar:?]
        at net.citizensnpcs.npc.CitizensNPC.update(CitizensNPC.java:283) [Citizens-2.0.24-b1628.jar:?]
        at net.citizensnpcs.nms.v1_13_R2.entity.WolfController$EntityWolfNPC.mobTick(WolfController.java:187) [Citizens-2.0.24-b1628.jar:?]
        at net.minecraft.server.v1_13_R2.EntityInsentient.doTick(EntityInsentient.java:665) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.EntityLiving.movementTick(EntityLiving.java:2254) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.EntityInsentient.movementTick(EntityInsentient.java:510) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.EntityAgeable.movementTick(EntityAgeable.java:145) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.EntityAnimal.movementTick(EntityAnimal.java:27) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.EntityWolf.movementTick(EntityWolf.java:142) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.EntityLiving.tick(EntityLiving.java:2098) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.EntityInsentient.tick(EntityInsentient.java:248) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.EntityWolf.tick(EntityWolf.java:157) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.World.entityJoinedWorld(World.java:1331) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.World.g(World.java:1298) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.World.tickEntities(World.java:1132) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.WorldServer.tickEntities(WorldServer.java:583) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:968) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:831) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:729) [spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NullPointerException
        at me.blackvein.quests.listeners.PlayerListener.onEntityDeath(PlayerListener.java:546) ~[?:?]
        at jdk.internal.reflect.GeneratedMethodAccessor328.invoke(Unknown Source) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:566) ~[?:?]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:312) ~[spigot-1.13.2.jar-2019-04-03-0645:git-Spigot-f09662d-0e1cea5]
        ... 34 more
```

**If applicable, can you provide an example quest from quests.yml _using Pastebin.com_?**
No active quests in quests.yml:
```
quests: {}
```

**Comment by PikaMug** [comment]
Use http://ci.ac3-servers.eu/job/Quests/717/ once available

**Comment by SlimeDog** [comment]
Confirmed fixed Quests-3.6.5-b717. Thanks.
